default_config.toml

```diff
@@ -30,5 +30,7 @@
 "music" = ""
 "disk usage" = ""
 ".pdf" = ""
+
 [other]
-"fallback_icon" = "🤨"
+fallback_icon = "🤨"
+merge = false
```

Comment:
I get what it's doing because I read the code and played around with it, but I think it's not obvious without more context.

Two suggestions:
* Could we use a more descriptive name? Maybe `deduplicate_window_names`
* Could we add a few words of documentation to explain the feature? Maybe something in the readme and something in the CLI app description?

Reply:
I didn't actually change the default, did I? I just copied it from default_config.toml

Reply:
Perhaps `deduplicate_icons`?

Reply:
Yes, that sounds better